frontend/src/metabase/query_builder/actions.js

```diff
@@ -298,6 +299,10 @@ export const initializeQB = (location, params) => {
     dispatch(resetQB());
     dispatch(cancelQuery());
 
+    // preload metadata that's used in DataSelector
+    dispatch(Databases.actions.fetchList({ include: "tables" }));
```

Comment:
Assuming that this fetch is async, how will Very Big Setups (many databases and even more tables in each database) affect usability?

Reply:
I suppose some computers could have some browser lag while processing the big blob of data. I think you'd need a very big db and a pretty slow computer to feel it though.

We have some a test db on stats with thousands of tables. This leads to a 150kb response for this call. Hopefully, that's small enough to avoid lag. I can't notice any myself, but I imagine some customers are on older hardware.